cve_bin_tool/output_engine/__init__.py

```diff
@@ -123,6 +123,7 @@ def output_pdf(
         merge_report,
         affected_versions: int = 0,
         exploits: bool = False,
+        all_product_data=None,
```

Comment:
I'm surprised black didn't change this one to have spaces like the others above, but I guess if it didn't it didn't.